### ISSUE TITLE
RPi2 - "Plex is not Reachable" error on v 1.0.2.89

### ISSUE THREAD

**Issue by cscheib** [plexinc/plex-media-player#157]
It seems wired Ethernet is not coming up properly on v 1.0.2.89 on my Raspberry Pi 2.  After booting, all that is displayed on the screen is "Plex is not Reachable" (with a little bit of other text - I'll grab that in a few and edit this).  So, seemingly, the network isn't coming up properly for me.

The link and activity lights seem to be working fine on the RPi2.

I am able to make PMP 1.0.1.42 and 1.0.0.4 work on the same piece of hardware.  I have swapped SD cards back to 1.0.1.42 in the interim.

**Comment by cscheib** [comment]
And even weirder... after multiple reboots, power down/ups, and a re-flash of the card, I was receiving the error... but, after switching back to the card that has 1.0.1.42 to verify it was still operational, then putting the 1.0.2.89 card back in, it seems to work fine

**Comment by BrianGilbert** [comment]
Confirming this issue.

**Comment by tru** [comment]
Can you try the 1.0.3 RC?

**Comment by cscheib** [comment]
@tru is there a pre-built image available somewhere of 1.0.3 for RPi2?

**Comment by BrianGilbert** [comment]
Confirming this is resolved in the latest release version: PlexMediaPlayer-1.0.3.140-16fa0ecc.RPi2-arm.img.gz

**Comment by lynndixon** [comment]
Im getting this same error on the version available from the Plex.tv downloads section:  Version 1.0.3.140-16fa0ecc

**Comment by mmccollum2** [comment]
Hi all, 

Came across this thread with the same problem, i was using 1.0.0.4-53192cb0 since the end of October when it came out on my RPi2. Tried to run 1.0.3.140-16fa0ecc on a new card and it came up with the same error of "Plex is not reachable" even though the wired network is coming up just fine at bootup on dhcp and accessible with ssh (root/plex). I checked the plex log files (tail -f /storage/.local/share/plexmediaplayer/logs/plexmediaplayer.log) while it was booting up and it was like it was trying to access an empty list of servers and since it couldn't connect to any of them, it just fails. well after trying to figure out why, i just waited and the "Retry, Go Home, Sign in" options showed up on the bottom, Sign in, do your pin-link and it works fine.

TL;DR - Fixed for me. - Wait a few minutes if it says unreachable, and options will show up at the bottom!

Just my experience though!

**Comment by cscheib** [comment]
I'm no longer using RPi2 for this.  Closing issue just for cleanup on my end.
